core/accounting.go

```diff
@@ -0,0 +1,72 @@
+package core
+
+import (
+	"math/big"
+	"sync"
+	"time"
+)
+
+// Use transcodeLoopTimeout as TTL for a ManifestID's credit balance
+// We should prevent the use of global state if they are not constants
```

Comment:
What's the relevance of this line to the previous line? Or is this a general comment

Reply:
Just a general comment, Can be removed. It's also not correct and should read "We should prevent the use of package level variables if they are not constants"